pyo3-macros-backend/src/module.rs

```diff
@@ -395,7 +392,11 @@ fn process_functions_in_module(options: &PyModuleOptions, func: &mut syn::ItemFn
                 let name = &func.sig.ident;
                 let statements: Vec<syn::Stmt> = syn::parse_quote! {
                     #wrapped_function
-                    #module_name.as_borrowed().add_function(#pyo3_path::wrap_pyfunction!(#name, #module_name.as_borrowed())?)?;
+                    {
+                        #[allow(unknown_lints, unused_imports, redundant_imports)]
+                        use #pyo3_path::{PyNativeType, types::PyModuleMethods};
```

Comment:
Doesn't this bloat the generated code as we will generate significantly more token? (Bloat meaning compile time as this certainly should not have a runtime effect.)

Reply:
I don't think this matters.